README.md

```diff
@@ -51,6 +51,27 @@ $ cd src
 $ PKG_CONFIG_PATH=/build/root/lib64/pkgconfig DESTDIR=/build/root make install
 ```
 
+Distributions
+=====
+
+Distributions packaging libbpf from this mirror:
+  - [Fedora](https://src.fedoraproject.org/rpms/libbpf)
+  - [Gentoo](https://packages.gentoo.org/packages/dev-libs/libbpf)
+
+Benefits of packaging from the mirror over packaging from kernel sources:
+  - Consistent, ABI matching versioning across distributions.
```

Comment:
this is  a bit hard to read, how about just:
```
Consistent versioning across distributions;
```

ABI guarantees are independent of packaging and every libbpf commit should preserve it, so it's not really Github-related